src/status_im/ethereum/ens.cljs

```diff
@@ -35,15 +35,17 @@
                   :on-error   #(cb "0x")}))
 
 (defn pubkey
-  [chain-id ens-name cb]
-  {:pre [(is-valid-eth-name? ens-name)]}
-  (json-rpc/call {:method     "ens_publicKeyOf"
-                  :params     [chain-id ens-name]
-                  :on-success cb
-                  ;;at some point infura started to return execution reverted error instead of "0x"
-                  ;;result
-                  ;;our code expects "0x" result
-                  :on-error   #(cb "0x")}))
+  ([chain-id ens-name on-success on-error]
+   {:pre [(is-valid-eth-name? ens-name)]}
+   (json-rpc/call {:method     "ens_publicKeyOf"
+                   :params     [chain-id ens-name]
+                   :on-success on-success
+                   :on-error   on-error}))
+  ;;at some point infura started to return execution reverted error instead of "0x"
```

Comment:
I know it wasn't you who wrote it, but since you're touching this function, could you please reword this comment, or at least break the lines correctly?

Reply:
done